mne/viz/tests/test_3d.py

```diff
@@ -74,6 +76,8 @@
   0.1250  0.750e-03  0.750e-03  0.750e-03  0.000  0.000  1.000
 """
 
+rng = np.random.RandomState(0)
```

Comment:
needed?

Reply:
@agramfort This code has been copied and modified from somewhere else in MNE. rng is used in test_mixed_sources_plot_surface() to generate random number. I could use np.random.seed(seed=0) and np.random.randn() instead, if you prefer. It has the advantage of not adding "global" code and having all this locally in test_mixed_sources_plot_surface. I am not sure about the advantages of using the Mersenne Twister pseudo-random number generator in numpy.random.RandomState. As I said, I just copied this code from elsewhere in MNE, without questionning much why this generator had been chosen.

Reply:
Agreed better to have it than not. But better to nest it in the test function that actually uses it (clearer where it's used, and also no temptation to use it in other functions, which could change the result in the test you're running, test order can break things, etc.)

Reply:
Can you nest the `rng` def inside the function where it's used?

That plus add an entry to the BUG section of `doc/changes/latest.inc` and we should be good to go

Reply:
@larsoner Both are done by 9de82bc